moveit_commander/setup.py

```diff
@@ -1,8 +0,0 @@
-from setuptools import setup
```

Comment:
This is part of a package that is COLCON_IGNOREd right now, should we keep so it can be removed, when the package is ported/refactored/removed?